.travis.yml

```diff
@@ -1,3 +1,6 @@
+git:
+  depth: 3
```

Comment:
This might possibly break the master check (not `TRAVIS_PULL_REQUEST_SHA`) after pushes, which runs the full check between the first change, which might be older than 3 commits.